### PR TITLE
added HP69

### PR DESCRIPTION
**Pull request by AnanyaKirti**
added firmware, keymap and via support for HP69

## Description

added firmware, keymap and via support for HP69

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/HP69/HP69.h [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+) { \
+    { K00,   K01,   K02,   K03,   K04,   K05,   K06,   K07,   K08,   K09,   K0A,   K0B,   K0C,   K0D,   K0E   }, \
+    { K10,   K11,   K12,   K13,   K14,   K15,   K16,   K17,   K18,   K19,   K1A,   K1B,   K1C,   K1D,   K1E   }, \
+    { K20,   K21,   K22,   K23,   K24,   K25,   K26,   K27,   K28,   K29,   K2A,   K2B,   K2C,   KC_NO, K2E   }, \
+    { K30,   K31,   K32,   K33,   K34,   K35,   K36,   K37,   K38,   K39,   K3A,   K3B,   K3C,   K3D,   K3E   }, \
+    { K40,   K41,   K42,   KC_NO, KC_NO, KC_NO, K46,   KC_NO, KC_NO, K49,   K4A,   K4B,   K4C,   K4D,   K4E   }, \
+}
+
+// generated by KBFirmware JSON to QMK Parser
+// https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (ridingqwerty) [inline keyboards/HP69/HP69.h:35]:
```suggestion
```

File: keyboards/HP69/hp69(kbfirmware).json [hunk 2]
```diff
@@ -0,0 +1 @@
...egend":"→","state":{"x":16.25,"y":4,"r":0,"rx":0,"ry":0,"w":1,"h":1,"x2":0,"y2":0,"w2":0,"h2":0,"c":"#aca693","l":true,"a":4},"row":4,"col":13,"keycodes":[{"id":"KC_RGHT","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]},{"id":"KC_TRNS","fields":[]}]}],"controller":1,"bounds":{"min":{"x":0,"y":0},"max":{"x":17.25,"y":5}},"rows":5,"cols":15,"pins":{"row":["B0","B1","B2","B3","B4"],"col":["B5","B6","B7","C0","C1","C2","C3","C4","C5","C6","C7","D0","D1","D2","D3"],"num":null,"caps":null,"scroll":null,"compose":null,"kana":null,"led":null,"rgb":null},"macros":{},"quantum":"void matrix_init_user(void) {\n}\n\nvoid matrix_scan_user(void) {\n}\n\nbool process_record_user(uint16_t keycode, keyrecord_t *record) {\n\treturn true;\n}","settings":{"diodeDirection":0,"name":"HP69","bootloaderSize":2,"rgbNum":0,"backlightLevels":3}}}
```
Comment (ridingqwerty) [inline keyboards/HP69/hp69(kbfirmware).json:1]: This file should be deleted.

File: keyboards/HP69/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    ),
+    [3] = LAYOUT(
+        _______,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,   _______, _______,
+        _______,     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,        _______, _______,
+        _______,       _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______,            _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,        _______,   _______,
+        _______,     _______,   _______,                            _______,                                 _______,     _______,      _______, _______, _______
+    ),
+
+
```
Comment (ridingqwerty) [inline keyboards/HP69/keymaps/default/keymap.c:48]:
```suggestion
```

These layers can be removed for the default keymap as they are superfluous and inaccessible.

File: keyboards/HP69/keymaps/via/rules.mk [hunk 4]
```diff
@@ -0,0 +1,7 @@
+VIA_ENABLE = yes
+
+BOOTMAGIC_ENABLE = lite
+
+MOUSEKEY_ENABLE = no
+
```
Comment (ridingqwerty) [inline keyboards/HP69/keymaps/via/rules.mk:6]:
```suggestion
BOOTMAGIC_ENABLE = lite
MOUSEKEY_ENABLE = no
```

File: keyboards/HP69/rules.mk [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+
+# generated by KBFirmware JSON to QMK Parser
+# https://noroadsleft.github.io/kbf_qmk_converter/
```
Comment (ridingqwerty) [inline keyboards/HP69/rules.mk:28]:
```suggestion
```

**Review by ridingqwerty — CHANGES_REQUESTED** [review]
Some minor cleanup suggested.

**Comment by AnanyaKirti** [comment]
hey, cleaned up as requested, please check again, thanks

File: keyboards/HP69/hp69(kbfirmware).json (hunk 2 above)
Comment (AnanyaKirti) [reply keyboards/HP69/hp69(kbfirmware).json]: deleted

File: keyboards/HP69/config.h [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x416B
+#define PRODUCT_ID      0x0001
+#define DEVICE_VER      0x0011
+#define MANUFACTURER    Desiboards
+#define PRODUCT         HP69
+#define DESCRIPTION     A 69 layout keyboard with apple bottom row
```
Comment (ridingqwerty) [inline keyboards/HP69/config.h:28]:
```suggestion
```

**Review by ridingqwerty — CHANGES_REQUESTED** [review]
Another small fixup regarding removing the `DESCRIPTION` field in config.h.

Also, per the submission checklist, all directories (`HP69`) and files (`HP69.{c,h}`) should be lowercase -- sorry for missing this the first time around.

File: keyboards/HP69/config.h (hunk 6 above)
Comment (AnanyaKirti) [reply keyboards/HP69/config.h]: removed in lastest commit along with name changed to lowercase

**Comment by AnanyaKirti** [comment]
hey, cleaned up as requested, please check again. Renamed the keyboard to hp69, from HP69, sorry about that

**Comment by ridingqwerty** [comment]
The `#include`s and build instructions now have lowercase `hp69`, but the `HP69` directory and `HP69.{c,h}` filenames themselves are still uppercase.  Let me know if you need help correcting that.

**Comment by AnanyaKirti** [comment]
> The `#include`s and build instructions now have lowercase `hp69`, but the `HP69` directory and `HP69.{c,h}` filenames themselves are still uppercase. Let me know if you need help correcting that.

Please help me, I've changed it to hp69 on my device and pushed it

**Comment by ridingqwerty** [comment]
Hi @AnanyaKirti, sorry for the delay.

Here's what I did after pulling your branch down (commands run from root QMK directory):

```
mv keyboards/HP69/ keyboards/hp69/
mv keyboards/hp69/HP69.c keyboards/hp69/hp69.c
mv keyboards/hp69/HP69.h keyboards/hp69/hp69.h
```

After that did the usual add/commit/push to the branch being PR'd.  Let me know if you need any further help here.

**Comment by AnanyaKirti** [comment]
opened a new one, couldn't change the name, client shows lowercase but GitHub.com won't show the changes

**Pull request closed without merging**